src/packcc.c

```diff
@@ -3134,20 +3142,17 @@ static bool_t generate(context_t *ctx) {
             "    size_t end;\n"
             "} pcc_range_t;\n"
             "\n",
-            stream
-        );
+            stream);
         fprintf_e(
             stream,
             "typedef %s%spcc_value_t;\n"
             "\n",
-            vt, vp ? "" : " "
-        );
```

Comment:
I really like the original style with unindented parenthesis on separate line for multiline arguments. But so far I could not find a configuration that would keep them...